### PR TITLE
Fixes problem authenticating with bidder number / phone number

### PR DESCRIPTION
**Pull request by ashfurrow**
The [issue](https://trello.com/c/6IZJd0oU/252-showstopper-kiosk-users-who-enter-bidder-number-should-be-asked-for-pin-and-not-email-address) was caused by a change to Moya's change in error-handling that I missed.

I took a look around for other places using the old logic, found one, but it was only related to logging. 

Oh, also added an admin button to bid the next increment of a sale artwork:

screen shot 2016-02-04 at 8 45 36 pm

Should speed up debugging :smiley_cat:

### CODE REVIEW

File: Kiosk/App/SwiftExtensions.swift [hunk 1]
```diff
@@ -54,3 +54,19 @@ extension Optional where Wrapped: Occupiable {
         return !isNilOrEmpty
     }
 }
+
+// TODO: PR this into Moya
```
Comment (ashfurrow) [inline Kiosk/App/SwiftExtensions.swift]: I have so PR'd: https://github.com/Moya/Moya/pull/401

However, we're a major version of Moya behind, and I don't have time to update right now. We'll put this on the back-burner for now.

**Comment by ashfurrow** [comment]
Nice! I was hoping that would fix the CI :joy_cat:

File: Kiosk/App/SwiftExtensions.swift (hunk 1 above)
Comment (acjay) [inline Kiosk/App/SwiftExtensions.swift]: Amused that we're a version behind on the library you started :) :)

File: Kiosk/Bid Fulfillment/ConfirmYourBidViewController.swift [hunk 2]
```diff
@@ -83,7 +83,11 @@ class ConfirmYourBidViewController: UIViewController {
                     // error to see if it's the original URL to know if the
                     // request succeeded.
 
-                    let response = (error as NSError).userInfo["data"] as? Moya.Response
+                    var response: Moya.Response?
```
Comment (orta) [inline Kiosk/Bid Fulfillment/ConfirmYourBidViewController.swift]: if it's lazy set once I think you can use a `let` here.

**Comment by orta** [comment]
Cool, looks good.

File: Kiosk/Bid Fulfillment/ConfirmYourBidViewController.swift (hunk 2 above)
Comment (ashfurrow) [inline Kiosk/Bid Fulfillment/ConfirmYourBidViewController.swift]: Only if all code paths set it exactly once, I'm afraid.

Comment (ashfurrow) [inline Kiosk/Bid Fulfillment/ConfirmYourBidViewController.swift]: (We might not set it at all, in this case.)

**Pull request merged**